### PR TITLE
Error Messages + Code Cleanup

### PR DESCRIPTION
**Pull request by ColtenVanTussenbrook**
### Description
https://moderntribe.atlassian.net/browse/FRAME-30
- adding error messages to:
-- checkbox
-- dropdown
-- file upload
-- password
-- radio button group
-- text field
- errors should show above helper text and display as red
- new ErrorText component
- changing commas to semi colons in interfaces (thanks TS)
- fixing import semantics

### Screencast
- http://p.tri.be/8gAwWL

### CODE REVIEW

File: src/components/checkbox/checkbox.tsx [hunk 1]
```diff
@@ -29,10 +32,27 @@ const StyledCheckbox = styled(MuiCheckbox, {
...
+    <FormGroup>
+      {
+        inputLabel
+        && <InputTitle>{inputLabel}</InputTitle>
+      }
+      <FormControlLabel control={<StyledCheckbox />} {...rest} />
+      {
+        error
+        && <ErrorText sx={{ marginTop: 0 }}>{errorMessage}</ErrorText>
+      }
```
Comment (chumpgrub) [inline src/components/checkbox/checkbox.tsx:53]: I think we should check for `errorMessage` here, too, so as not to render the empty component.

**Review by chumpgrub — CHANGES_REQUESTED** [review]
I dig it! Although, I think we should be checking for both `error` and `errorMessage` before rendering the `ErrorText` component. Probably a fringe case, but it could prevent content shifts in the dom.

File: src/components/checkbox/checkbox.tsx (hunk 1 above)
Comment (ColtenVanTussenbrook) [reply src/components/checkbox/checkbox.tsx]: That's a good call, I added that check for `errorMessage`

**Review by chumpgrub — APPROVED** [review]
Let's goooo!

**Pull request merged**